curated_transformers/models/falcon/config.py

```diff
@@ -23,8 +23,8 @@ def __init__(
         *,
         dropout_prob: float = 0.1,
         hidden_width: int = 2560,
-        multi_query: bool = True,
-        num_attention_heads: int = 71,
+        num_query_heads: int = 71,
+        num_kv_heads: int = 1,
```

Comment:
Why `kv` and not `key_value`? We're mapping them anyway.

Reply:
I renamed them later, forgot these. Fixed.